spec/requests/api/v1/documents_spec.rb

```diff
@@ -5,9 +5,14 @@
   let(:id) { "some really/weird@id.name" }
   let(:valid_params) do
     {
-     "document_id" => id, "title" => "my title", "path" => "http://www.gov.gov/goo.html",
-     "description" => "my desc", "promote" => true,
-     "language" => 'hy', "content" => "my content", "tags" => "Foo, Bar blat",
+      "document_id" => id,
```

Comment:
I wasn't going to say anything about this, and I'm glad you flattened this out :)